apps/server/src/modules/user-migration/service/user-migration.service.ts

```diff
@@ -6,19 +6,28 @@ import { EntityNotFoundError } from '@shared/common';
 import { SystemDto, SystemService } from '@src/modules/system/service';
 import { PageTypes } from '../interface/page-types.enum';
 import { PageContentDto } from './dto/page-content.dto';
+import { UserService } from '../../user';
```

Comment:
Please check your imports. Here you should import an external service with relative path @src/...

Reply:
yes, thanks :)